<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_emp</class>
 <widget class="QDialog" name="add_emp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Employee ID:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Employee Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>150</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Date Of Birth:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Employee Qualifications:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="emp_id">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="emp_name">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="emp_dob">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>140</y>
     <width>110</width>
     <height>26</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>dd/MM/yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="role">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>190</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Technician</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Cloud Architect</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Junior Manager</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Senior Manager</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="emp_add">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>250</y>
     <width>83</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
